core/tests/conftest.py

```diff
@@ -1,5 +1,44 @@
-from django.conf import settings
-from django.core.management import call_command
+import shutil
+import tempfile
 
-def pytest_configure(config):
-    settings.STORAGES['staticfiles']['BACKEND'] = 'django.contrib.staticfiles.storage.StaticFilesStorage'
+import pytest
+from django.contrib.auth import get_user_model
+
+from core.models import Profile, Project
+
+User = get_user_model()
+
+
+@pytest.fixture
+def user(db):
+    return User.objects.create_user(username="testuser", email="test@example.com", password="testpass123")
+
+
+@pytest.fixture
+def profile(user):
+    return Profile.objects.create(
+        user=user,
+    )
+
+
+@pytest.fixture
+def project(profile):
+    return Project.objects.create(
+        profile=profile, name="Test Project", slug="test-project", url="https://example.com", public=True
+    )
```

Comment:
@rasulkireev I can recommend sparse fixtures for this, in particular:

- a) https://github.com/FactoryBoy/factory_boy
- b) https://github.com/model-bakers/model_bakery

With sparse fixtures, (simplified) you'll never have to care about dummy values in tests again. Just an idea.